wpilibc/src/main/native/include/frc/PneumaticsControlModule.h

```diff
@@ -13,23 +13,52 @@
 #include "PneumaticsBase.h"
 
 namespace frc {
+/** Module class for controlling a Cross The Road Electronics Pneumatics Control
+ * Module. */
 class PneumaticsControlModule : public PneumaticsBase {
  public:
+  /** Constructs a PneumaticsControlModule with the default id (0). */
   PneumaticsControlModule();
+
+  /**
+   * Constructs a PneumaticsControlModule.
+   *
+   * @param module module number to construct
+   */
   explicit PneumaticsControlModule(int module);
 
   ~PneumaticsControlModule() override = default;
 
   bool GetCompressor() const override;
 
+  /**
+   * Disables the compressor. The compressor will not turn on until
+   * EnableCompressorDigital() is called.
+   */
   void DisableCompressor() override;
 
   void EnableCompressorDigital() override;
 
+  /**
+   * Enables the compressor in digital mode. Analog mode is unsupported by the
```

Comment:
Sure this isn't a no-op?

Reply:
Yeah this just forwards to EnableDigital on the CTRE stuff.